docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -40,14 +40,19 @@
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
 <h2>Release notes: Open GEE 5.3.1</h2>
 
-  <h3 id="overview_5.3.1">New Features</h3>
-  <p><strong>Uninstallation scripts are now copied to <code>/opt/google/install</code>. </strong>This will allow the user to uninstall without having to have a copy of the source code on hand.</p>
-  <p><strong>Memory and performance improvements for large builds.</strong> Memory utilization and performance have been improved, especially for large builds.</p>
-  <p><strong>Increased logging during long Fusion operations.</strong> Fusion can now be configured to log progress while calculating overlaps between resources.</p>
-  <p><strong>Overlay option is now supported in terrain project command-line interfaces.</strong> <code>genewterrainproject</code>, <code>geaddterrainproject</code> and <code>gemodifyterrainproject</code> have a new option for overlay support.</p>
-  <p><strong>Cutter embeds extra metadata into portable files.</strong>  The bounding box metadata can be used to improve loading performance in some applications.</p>
-  <h3 id="supported_5.3.1">Supported Platforms</h3>
+<p>Open GEE 5.3.1 is currently in development</p>
 
+  <h5>
```

Comment:
Changes that is already in [5.3.1 release notes doc](https://github.com/google/earthenterprise/blob/master/docs/geedocs/5.3.1/answer/7160008.html) needs to be copied into [same release notes doc stored under 5.3.2 folder](https://github.com/google/earthenterprise/blob/master/docs/geedocs/5.3.2/answer/7160008.html). The contents needs to be the same except the table of contents.